addons/bestja_estimation_reports/views/estimation_reports_summary.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record model="ir.ui.view" id="estimation_report_entry_organization_graph">
+            <field name="name">bestja.estimation_report_entry.graph</field>
+            <field name="model">bestja.estimation_report_entry</field>
+            <field name="arch" type="xml">
+                <graph type="pivot">
+                    <field name="organization"/>
+                    <field name="day" type="col" interval="day"/>
+                    <field name="tonnage" type="measure"/>
+                </graph>
+            </field>
+        </record>
+        <record model="ir.actions.act_window" id="action_estimation_organization_summary">
+            <field name="name">Statystyki raportów</field>
+            <field name="res_model">bestja.estimation_report_entry</field>
+            <field name="view_mode">graph</field>
+            <field name="view_id" ref="estimation_report_entry_organization_graph"/>
+            <field name="domain">[
+                ('estimation_report.state', '!=', 'draft'),
+            ]</field>
+            <field name="context">{
+                'search_default_accepted': True,
+            }</field>
+        </record>
+
+        <record model="ir.ui.view" id="estimation_report_entry_store_graph">
+            <field name="name">bestja.estimation_report_entry.graph</field>
+            <field name="model">bestja.estimation_report_entry</field>
+            <field name="arch" type="xml">
+                <graph type="pivot">
+                    <field name="chain_name"/>
```

Comment:
Shouldn't just using `chain` and `store` instead of `chain_name` and `store_name` work fine?